recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ source:
 
 build:
   skip: true  # [win and py<35]
-  number: 1
+  number: 2
```

Comment:
It's changing the Windows package though. So I have to bump it to get a new package for Windows.